gpii/node_modules/gsettingsBridge/tests/gsettingsTests.js

```diff
@@ -209,6 +211,10 @@ jqUnit.test("Setting single keys via gsettings handler", function () {
     gpii.gsettings.setSingleKey("net.gpii.testing.gsettings.single-set", "string-setting", "Absolutely awesome");
     jqUnit.assertEquals("Checking 'string' key", "Absolutely awesome",
         gpii.gsettings.getSingleKey("net.gpii.testing.gsettings.single-set", "string-setting"));
+    // set and check non-existent key
+    gpii.gsettings.setSingleKey("net.gpii.testing.gsettings.single-set", "unicorn", "Mythical");
+    jqUnit.assertEquals("Checking non-existaent 'string' key", undefined,
```

Comment:
Why didn't `grunt lint` find that????  Perhaps we need to add a spell checker grunt task. ;-)